pkc

```diff
@@ -45,6 +45,11 @@ Options:
 EOF
 }
 
+escalate() {
```

Comment:
Maybe it should be called "elevate", because this is not privilege escalation.

Reply:
Yes! Good idea.